### PR TITLE
[thread-cert] refactor case 6.4.1 using pktverify

### PR DESCRIPTION
**Pull request by FiveDimensions**

### CODE REVIEW

File: tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py [hunk 1]
```diff
@@ -55,19 +80,128 @@ def test(self):
...
-        self.nodes[ED].start()
+        self.nodes[MTD].start()
         self.simulator.go(5)
-        self.assertEqual(self.nodes[ED].get_state(), 'child')
+        self.assertEqual(self.nodes[MTD].get_state(), 'child')
+
+        self.collect_ipaddrs()
+
+        dut_addr = self.nodes[MTD].get_ip6_address(config.ADDRESS_TYPE.LINK_LOCAL)
+        self.assertTrue(self.nodes[LEADER].ping(dut_addr, size=256))
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py:90]: avoid magic number, use `FRAGMENTED_DATA_LEN`?

File: tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py [hunk 2]
```diff
@@ -55,19 +80,128 @@ def test(self):
...
+        self.simulator.go(1)
+        self.assertTrue(self.nodes[LEADER].ping(dut_addr))
+        self.simulator.go(1)
+
+        self.assertTrue(self.nodes[LEADER].ping('ff32:40:fd00:db8::1', size=256))
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py:95]: use `LINK_LOCAL_All_THREAD_NODES_MULTICAST_ADDRESS`

File: tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py [hunk 3]
```diff
@@ -55,19 +80,128 @@ def test(self):
...
+        self.simulator.go(1)
+        self.assertTrue(self.nodes[LEADER].ping('ff32:40:fd00:db8::1'))
+        self.simulator.go(1)
+
+        if self.TOPOLOGY[MTD]['mode'] == 'rn':
+            self.assertTrue(self.nodes[LEADER].ping('ff02::1', size=256))
```
Comment (librasungirl) [inline tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py:101]: use `LINK_LOCAL_ALL_NODES_MULTICAST_ADDRESS`?

File: tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py (hunk 1 above)
Comment (FiveDimensions) [reply tests/scripts/thread-cert/Cert_6_4_01_LinkLocal.py:90]: sure

**Review by librasungirl — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**